presto-hive/src/main/java/com/facebook/presto/hive/HiveInputInfo.java

```diff
@@ -45,4 +53,10 @@ public boolean isTruncated()
     {
         return truncated;
     }
+
+    @JsonProperty
+    public List<DateTime> getLastAccessTimes()
+    {
+        return lastAccessTimes;
+    }
```

Comment:
Where is this method going to be used?

Reply:
For Depa logging for input tables/partitions.